ProjectLighthouse/Controllers/UserController.cs

```diff
@@ -24,18 +25,32 @@ public UserController(Database database)
             this.database = database;
         }
 
-        [HttpGet("user/{username}")]
-        public async Task<IActionResult> GetUser(string username)
+        public async Task<string> GetSerializedUser(string username)
         {
-            User user = await this.database.Users.Include(u => u.Location).FirstOrDefaultAsync(u => u.Username == username);
+            User? user = await this.database.Users.Include(u => u.Location).FirstOrDefaultAsync(u => u.Username == username);
+            return user?.Serialize();
+        }
 
+        [HttpGet("user/{username}")]
+        public async Task<IActionResult> GetUser(string username) {
+            string? user = await this.GetSerializedUser(username);
```

Comment:
This file should have a `#nullable enable` at the beginning of it if you use nullable types.

Reply:
Fixed in https://github.com/LBPUnion/project-lighthouse/pull/38/commits/45b8464d1cb676080d24e3cc5a4036ff65d804cb